x-pack/test/ftr_apis/security_and_spaces/apis/clean.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { USERS, User, ExpectedResponse } from '../../common/lib';
+import { FtrProviderContext } from '../services';
+import { createData, createTestSpaces, deleteData, deleteTestSpaces } from './test_utils';
+
+// eslint-disable-next-line import/no-default-export
```

Comment:
nit: we may want to edit `.eslintrc` to avoid these exceptions:
https://github.com/elastic/kibana/blob/d6be4a4b06921fd43b0800ab7b70805a3b015d4f/.eslintrc.js#L595